include/swift/AST/IRGenOptions.h

```diff
@@ -52,6 +52,7 @@ enum class IRGenDebugInfoKind : unsigned {
   LineTables, /// Line tables only.
   ASTTypes,   /// Line tables + AST type references.
   DwarfTypes, /// Line tables + AST type references + DWARF types.
+  CodeViewTypes, /// Line tables + AST type references + CodeView types.
```

Comment:
Thinking more about what @adrian-prantl said, I think it makes sense to split this out and crease `IRGenDWARFDebugInfoKind` and `IRGenCodeViewDebugInfoKind` and a separate switch between the two formats.

Reply:
I can make these changes, but rather than `IRGenCodeViewDebugInfoKind` I was thinking of making it `IRGenCodeViewDebugInfo` with only values `None` and `Yes`. Would it make more sense to make this a bool?

Reply:
Nah, err on the side of enums when it's not obvious which one would be the default in the long term. It makes it easier at call sites too.